qsiprep/tests/data/maternal_brain_project_outputs.txt

```diff
@@ -7,29 +7,29 @@ logs/CITATION.tex
 sub-01
 sub-01.html
 sub-01/anat
-sub-01/anat/sub-01_desc-aseg_dseg.nii.gz
-sub-01/anat/sub-01_desc-brain_mask.nii.gz
-sub-01/anat/sub-01_desc-preproc_T1w.json
-sub-01/anat/sub-01_desc-preproc_T1w.nii.gz
-sub-01/anat/sub-01_dseg.nii.gz
-sub-01/anat/sub-01_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.h5
-sub-01/anat/sub-01_from-T1wACPC_to-T1wNative_mode-image_xfm.mat
-sub-01/anat/sub-01_from-T1wNative_to-T1wACPC_mode-image_xfm.mat
-sub-01/anat/sub-01_from-T1w_to-MNI152NLin2009cAsym_mode-image_xfm.h5
+sub-01/anat/sub-01_from-ACPC_to-MNI152NLin2009cAsym_mode-image_xfm.h5
+sub-01/anat/sub-01_from-ACPC_to-anat_mode-image_xfm.mat
+sub-01/anat/sub-01_from-MNI152NLin2009cAsym_to-ACPC_mode-image_xfm.h5
+sub-01/anat/sub-01_from-anat_to-ACPC_mode-image_xfm.mat
+sub-01/anat/sub-01_space-ACPC_desc-aseg_dseg.nii.gz
+sub-01/anat/sub-01_space-ACPC_desc-brain_mask.nii.gz
+sub-01/anat/sub-01_space-ACPC_desc-preproc_T1w.json
+sub-01/anat/sub-01_space-ACPC_desc-preproc_T1w.nii.gz
+sub-01/anat/sub-01_space-ACPC_dseg.nii.gz
 sub-01/ses-01
 sub-01/ses-01/anat
-sub-01/ses-01/anat/sub-01_ses-01_rec-autobox_from-orig_to-T1w_mode-image_xfm.txt
+sub-01/ses-01/anat/sub-01_ses-01_rec-autobox_from-orig_to-anat_mode-image_xfm.txt
 sub-01/ses-01/dwi
 sub-01/ses-01/dwi/sub-01_ses-01_confounds.tsv
-sub-01/ses-01/dwi/sub-01_ses-01_desc-ImageQC_dwi.csv
-sub-01/ses-01/dwi/sub-01_ses-01_desc-SliceQC_dwi.json
 sub-01/ses-01/dwi/sub-01_ses-01_hmcOptimization.csv
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-3dSHORE_cnr.nii.gz
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-brain_mask.nii.gz
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.b
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.bval
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.bvec
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.nii.gz
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.json
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_desc-preproc_dwi.b_table.txt
-sub-01/ses-01/dwi/sub-01_ses-01_space-T1w_dwiref.nii.gz
+sub-01/ses-01/dwi/sub-01_ses-01_space-ACPC_desc-3dSHORE_cnr.nii.gz
```

Comment:
Just a note for the future- we might want to replace the `cnr` suffix with `stat-CNR_dwimap`.